### PR TITLE
Beta15

### PR DESCRIPTION
**Pull request by karaok491**
Do not merge until beta 15 is tagged

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -39,6 +42,12 @@
   "extra": {
     "flarum-extension": {
       "title": "FoF Best Answer",
+      "category": "feature",
+      "info": {
```
Comment (askvortsov1) [inline composer.json:46]: We actually redid this piece recently, to use more of the native composer fields: https://github.com/flarum/core/blob/master/src/Extension/Extension.php#L352

For an example of the new system, see pretty much any bundled extension, I updated all of them today.

File: extend.php [hunk 2]
```diff
@@ -50,20 +53,48 @@
...
+                return Carbon::createFromTimeString($discussion->best_answer_set_at)->format(DateTime::RFC3339);
+            }
 
-    function (Application $app) {
-        $app->register(ConsoleProvider::class);
-    },
+            return null;
+        }),
+
+    (new Extend\ApiSerializer(ForumSerializer::class))
```
Comment (askvortsov1) [inline extend.php:88]: For these, use the Settings extender's `serializeToForum` method

File: js/src/admin/index.js [hunk 3]
```diff
@@ -1,53 +1,63 @@
...
+        .registerPermission({
             icon: 'far fa-comment',
             label: app.translator.trans('fof-best-answer.admin.permissions.best_answer_not_own_discussion'),
-            permission: 'discussion.selectBestAnswerNotOwnDiscussion',
-        });
-    });
+            permission: 'discussion.selectBestAnswerNotOwnDiscussion'
+        }, 'reply')
+
+    // app.extensionSettings['fof-best-answer'] = () =>
```
Comment (askvortsov1) [inline js/src/admin/index.js:51]: This should be good to eliminate if it works locally

File: src/Console/NotifySchedule.php [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+use Flarum\Foundation\Paths;
+use Flarum\Settings\SettingsRepositoryInterface;
+use Illuminate\Console\Scheduling\Schedule;
+
+class NotifySchedule
+{
+    public function __invoke()
+    {
+        $schedule = app()->make(Schedule::class);
```
Comment (askvortsov1) [inline src/Console/NotifySchedule.php:22]: If possible, these should be injected in the constructor.

**Review by askvortsov1 — CHANGES_REQUESTED** [review]

File: js/src/admin/index.js (hunk 3 above)
Comment (karaok491) [reply js/src/admin/index.js:51]: Ooh, I left this comment to remind me to ask you.... `min`, `max` and `placeholder`, I couldn't see how to implement them using the new `registerSetting`?

File: composer.json (hunk 1 above)
Comment (karaok491) [reply composer.json:46]: Perfect, I'll update...

File: src/Console/NotifySchedule.php (hunk 4 above)
Comment (karaok491) [reply src/Console/NotifySchedule.php:22]: Will do!

**Comment by karaok491** [comment]
Closing broken PR - see #22

**Pull request closed without merging**